2-mandatory/4-restaurant.js

```diff
@@ -20,8 +20,22 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
-}
+ orderBurger: function(balance){
+  if(balance >= MENU.burger){
+    return balance - MENU.burger;
+  } else {
+    return balance;
+  }
+
+ },
+ orderFalafel: function(balance){
+  if(balance >= MENU.falafel){
+    return balance - MENU.falafel;
+  } else {
+    return balance;
+  }
+ }
+};
 
```

Comment:
Great job overall. keep going

Reply:
Hi @ellietms, thank you for reviewing my code and your suggestions. I keep them in mind.